logexp/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMG = gcr.io/google-containers/logexp
+TAG = v0.1.0
```

Comment:
Had that originally, but dropped it assuming that this would be a more stable image (not built regularly like the kubekins image probably) and could be used even outside test-infra.
Changed it now though, let's stick to that pattern. Thanks!